release/downloadIstioCtl.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+#
+# Separate downloader for istioctl
+#
+# This file will be fetched as: curl -L https://git.io/getLatestIstioCtl | sh -
+#
+INSTALL_DIR="$HOME/.istioctl/bin"
+
+# Determines the operating system.
+detect_os() {
+  OS=$(uname)
+  case $OS in
+    'Linux')
+      OSEXT='linux'
+      FILENAME='istioctl'
+      ;;
+    'WindowsNT')
```

Comment:
does it make sense to have a shell script download a windows binary? Presumably if they have bash then they also have WSL and can use the linux binary?

Reply:
A good point here. Perhaps we don't need to make the zip file available via the method? I like to hear what @linsun or @utka thinks about this suggestion.